src/static/riot/competitions/detail/_header.tag

```diff
@@ -0,0 +1,286 @@
+<comp-detail-header>
+    <div class="ui relaxed grid">
+        <div class="row">
+            <div class="three wide column">
+                <img class="ui medium circular image competition-image"
+                     alt="Competition Logo"
+                     src="{ competition.logo }">
+            </div>
+            <div class="ten wide column">
+                <div class="ui grid">
+                    <div class="row">
+                        <div class="column">
+                            <div class="competition-name underline">
+                                {competition.title}
+                            </div>
+                        </div>
+                    </div>
+                    <div if="{competition.admin}">
+                        <a href="{URLS.COMPETITION_EDIT(competition.id)}" class="ui button">Edit</a>
+                        <button class="ui small button" onclick="{show_modal.bind(this, '.manage-participants.modal')}">Participants</button>
+                        <button class="ui small button" onclick="{show_modal.bind(this, '.manage-submissions.modal')}">Submissions</button>
+                        <button class="ui small button" onclick="{show_modal.bind(this, '.manage-competition.modal')}">Dumps</button>
+                        <button class="ui small button" onclick="{show_modal.bind(this, '.migration.modal')}">Migrate</button>
+                    </div>
+                    <div class="row">
+                        <div class="column">
+                            <div>
+                                <span class="detail-label">Organized by:</span>
+                                <span class="detail-item">{competition.created_by}</span>
+                            </div>
+                            <div>
+                                <span class="detail-label">Current phase ends:</span>
+                                <span class="detail-item">{get_end_date(competition)}</span>
+                            </div>
+                            <div class="competition-secret-key" if="{ competition.admin }">
+                                <span class="secret-label">Secret url:</span>
+                                <span id="secret-url">https://{ URLS.SECRET_KEY_URL(competition.id, competition.secret_key) }</span>
+                                <span onclick="{copy_secret_url}" class="ui send-pop" data-content="Copied!"><i
+                                    class="ui copy icon"></i></span>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+            </div>
+            <div class="three wide column">
+                <div class="stat-buttons">
```

Comment:
eventually, we need to turn off the cursor: pointer and hover on these buttons so they don't look clickable, since they arent.. or we need to find a different semantic element to get similar styling. Did not address it as this was an issue before I re-did the header, and I didn't wanna mess with the styling for forever.

Reply:
turning off cursor pointer here should be relatively simple? can make an issue for it?

Reply:
issue created